pkg/identity/numericidentity.go

```diff
@@ -107,6 +108,13 @@ const (
 	ReservedCiliumEtcdOperator NumericIdentity = 107
 )
 
+var (
+	// localNodeMutex protects localNodeIdentity
+	localNodeMutex lock.Mutex
+	// localNodeIdentity is the endpoint identity allocated for the local node
+	localNodeIdentity = ReservedIdentityRemoteNode
```

Comment:
Nit: make this an anonymous struct type to keep the mutex together with the value it protects:

```suggestion
+// localNodeIdentity is the endpoint identity allocated for the local node
var localNodeIdentity = struct {
       lock.Mutex
       identity NumericIdentity
}{
       identity: ReservedIdentityRemoteNode,
}
```

Reply:
Much nicer, thanks!